### ISSUE TITLE
Show-HPOVFirmwarereport is not working. Version 3.0

### ISSUE THREAD

**Issue by mickechristenson** [HewlettPackard/POSH-HPOneView#215]
Please fill in as much information as possible to help resolve your issue.

### Expected Behavior
Get a list of the firmware on one server or many or for one enclosure.

### Actual Behavior
``` PowerShell
PS C:\WINDOWS\system32> Get-HPOVEnclosure -name 'ch002dc01' | Show-HPOVFirmwareReport

Show-HPOVFirmwareReport : A positional parameter cannot be found that accepts argument '@{type=EnclosureV300; scopeUris=System.Object[]; supportData
CollectionType=; remoteSupportUri=/rest/support/enclosures/09CZ303427FE; supportDataCollectionsUri=/rest/support/data-collections?deviceID=09CZ30342
7FE&category=enclosures; supportState=Enabled; supportDataCollectionState=; remoteSupportSettings=; fanBays=System.Object[]; uidState=Off; managerBa
ys=System.Object[]; powerSupplyBayCount=6; powerSupplyBays=System.Object[]; reconfigurationState=NotReapplyingConfiguration; deviceBayCount=16; uuid
=09CZ303427FE; enclosureGroupUri=/rest/enclosure-groups/0d755b68-9b95-4e1a-8545-49d4f5186ecc; logicalEnclosureUri=/rest/logical-enclosures/93730c1d-
38ca-4145-b0b9-3bbfb7f13edf; enclosureModel=BladeSystem c7000 Enclosure G2; enclosureTypeUri=/rest/enclosure-types/c7000; fanBayCount=10; forceInsta
llFirmware=False; fwBaselineName=HP Service Pack for ProLiant, 2015.10.0; fwBaselineUri=/rest/firmware-drivers/843216_001_spp_2015_10_0-SPP2015100_2
015_0921_6; interconnectBayCount=8; interconnectBays=System.Object[]; isFwManaged=True; enclosureType=C7000; partNumber=507019-B21; deviceBays=Syste
m.Object[]; licensingIntent=OneViewNoiLO; serialNumber=CZ303427FE; refreshState=NotRefreshing; status=OK; uri=/rest/enclosures/09CZ303427FE; name=CH
002DC01; state=Configured; stateReason=None; description=; created=2015-10-03T22:11:40.532Z; eTag=2017-01-12T11:32:39.396Z; modified=2017-01-12T11:3
2:39.396Z; category=enclosures; vcmDomainId=@d027f999b3fa83a0fdc18a327d89e0; vcmDomainName=OneViewDomain; vcmMode=True; vcmUrl=https://172.19.132.3;
 standbyOaPreferredIP=10.0.13.12; activeOaPreferredIP=10.0.13.11; assetTag=; rackName=AR07; oaBays=2; ApplianceConnection=}'.
At line:1 char:39
+ Get-HPOVEnclosure -name 'ch002dc01' | Show-HPOVFirmwareReport
+                                       ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Show-HPOVFirmwareReport], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,Show-HPOVFirmwareReport
```

### Steps to reproduce
``` PowerShell
PS C:\> Get-HPOVServer -Name xxxxxxx | Show-HPOVFirmwareReport
```

### Version Information
``` PowerShell
HPE OneView PowerShell Library Version (`Get-HPOVVersion`):3.0.1128.2242
HPE OneView Appliance Version:  3.00.05
```
Output from `$PSVersionTable` on your Windows Host:
``` PowerShell
PS C:\WINDOWS\system32> $PSVersionTable
PS C:\WINDOWS\system32> $PSVersionTable
Name                           Value                                                                                                                
----                           -----                                                                                                                
PSVersion                      5.0.10586.494                                                                                                        
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                              
BuildVersion                   10.0.10586.494                                                                                                       
CLRVersion                     4.0.30319.42000                                                                                                      
WSManStackVersion              3.0                                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                                  
SerializationVersion           1.1.0.1
```

**Comment by ChrisLynchHPE** [comment]
Thanks.  I tested this Cmdlet when I responded to your thread in the HPE OneView Communities.  I found and have fixed the issue that is causing the problem.  It will be fixed in the next update.

**Comment by mickechristenson** [comment]
Thanks a lot, Chris!

/Mikael

**Comment by ChrisLynchHPE** [comment]
This is fixed in [Release 3.0.1210.3013](https://github.com/HewlettPackard/POSH-HPOneView/releases/tag/v3.0.1210.3013).

Closing.

**Comment by thomaspunktelf** [comment]
Hi Chris,
I got a similar issue with the latest library release. 

PS C:\Thomas\IFAG\GUI> Get-HPOVServer -Name gxxxx004l | Show-HPOVFirmwareReport
Show-HPOVFirmwareReport : A parameter cannot be found that matches parameter name 'ProgressID'.
At line:1 char:34
+ Get-HPOVServer -Name gxxxx004l | Show-HPOVFirmwareReport
+  
    + CategoryInfo          : InvalidArgument: (:) [Show-HPOVFirmwareReport], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Show-HPOVFirmwareReport

PS C:\> Get-HPOVVersion


PowerShellLibraryVersion : 3.0.1293.3770
LibraryPath              : C:\Program Files\WindowsPowerShell\Modules\HPOneView.300



PS C:\> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.14393.1066
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.1066
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

Best regards
Tom
